app/src/pages/home.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { useState, useEffect, useContext } from 'react';
+
+import CircularProgress from '@mui/material/CircularProgress';
+import Paper from '@mui/material/Paper';
+import Snackbar from '@mui/material/Snackbar';
+import Button from '@mui/material/Button';
+
+import app from 'STARTER/feathers-client';
+import Login from 'STARTER/components/login';
+import Registration from 'STARTER/components/registration';
+import { AuthContext, AuthContextType, authData } from 'STARTER/contexts/auth'
+
+import { isMobile } from 'STARTER/utils';
+import '@shopify/polaris/build/esm/styles.css';
+import translations from '@shopify/polaris/locales/en.json';
+import { AppProvider } from '@shopify/polaris';
+
+export default function Home() {
+	const [snackBarOpen, setSnackBarOpen] = useState(false);
+	const [snackBarMessage, setSnackBarMessage] = useState('');
+
+  const { authenticate, isLoginLoading, isAuthed, logout } = useContext(AuthContext) as AuthContextType
+
+	const handleCloseSnackBar = () => setSnackBarOpen(false);
+
+	const onMobile = isMobile();
+
+	const textStyle = {
+		fontFamily: 'Roboto, Arial, Helvetica, sans-serif',
+		fontSize: 22,
+		fontWeight: 100,
+	};
+
+	return (
+		<AppProvider i18n={translations}>
```

Comment:
Maybe you're already working on this...but is it possible to include more of the translation code from IMIN in the starter project? It would be great to have an example already of how to switch languages and the code that highlights missing translations in yellow.

Reply:
update: commit has been pushed so the translations should now be working!

Reply:
Amazing! Thanks so much! this will be helpful going forward with new projects :)